README.md

```diff
@@ -55,28 +55,55 @@ getUser();
 ```
 **[⬆ back to top](#table-of-contents)**
 
-### Use searchable names
+### Use searchable names (part 1)
+
 We will read more code than we will ever write. It's important that the code we do write is 
 readable and searchable. By *not* naming variables that end up being meaningful for 
 understanding our program, we hurt our readers.
 Make your names searchable.
 
 **Bad:**
+
 ```php
-// What the heck is 86400 for?
-addExpireAt(86400);
+// What the heck is 448 for?
+$result = $serializer->serialize($data, 448);
+```
+
+**Good:**
 
+```php
+$json = $serializer->serialize($data, JSON_UNESCAPED_SLASHES | JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
 ```
 
-**Good**:
+### Use searchable names (part 2)
+
+**Bad:**
+
 ```php
-// Declare them as capitalized `const` globals.
-interface DateGlobal {
-    const SECONDS_IN_A_DAY = 86400;
+// What the heck is 4 for?
+if ($user->access & 4) {
+    // ...
+}
+```
+
+**Good:**
+
+```php
+class User
+{
+    const ACCESS_NONE = 0;
+    const ACCESS_READ = 1;
+    const ACCESS_CREATE = 2;
```

Comment:
Thanks for simplification.

Could you keep minimum possible constants here? 6 is quite a lot, moreoever with 4 repeated in last one.

I guess 3 would be fine:

E.g. NONE | READ |  WRITE

It would make it easier to read and focus on "move to constant" meaning instead "read all possible access rights".